src/main/resources/db/migration/data/R__2_3_1__REFERENCES_CODES_alerts.sql

```diff
@@ -0,0 +1,102 @@
+INSERT INTO REFERENCE_CODES (DOMAIN, CODE, PARENT_DOMAIN, PARENT_CODE, DESCRIPTION, LIST_SEQ, ACTIVE_FLAG, SYSTEM_DATA_FLAG, EXPIRED_DATE) VALUES ('ALERT', 'A', null, null, 'Social Care', 12, 'Y', 'N', null);
```

Comment:
trying this out by grouping reference data in different files so they get so big